RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, and where we’d like to grow in the future. 
+
+
+## Core Strategy Statement  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## Who we are
+
+We are a community of technical folk. Writers, developers, editors and more. We are passionate about technical writing, documentation and open source.
+
+
+### Our Audience
+
+Our primary audience includes:
+
+* Users of our products
+* Members of our community
+* People seeking to learn more about the technical writing domain
+
+
+### Our friends
+
+We are friends with (and many of us are active in) the [Write The Docs community](https://www.writethedocs.org/). 
+We are not competing with that community. We acknowledge that our interests overlap and that sometimes our activities align. 
+The relationship between our projects is mutually supportive and friendly.
+
+We welcome collaboration and sponsorship opportunities. We like to participate in writing programs and open source programs (like the Google Season of Docs).
+
+
+## What we do
+
+First and foremost, we create documentation templates with supporting meta-documents that help people use and understand the templates. As part of this work, we engage in many other initiatives around docs advocacy, docs education, and docs tooling. 
```

Comment:
+1. Something like "guides, blog posts, etc." would allow it flexibility.

I'd also maybe go with "Our primary product is our collection of templates," but that's also me pushing to delineate product material and production mechanisms with website material and publish mechanisms

Reply:
Thanks @barbaricyawps I like "guides" better, will change.
I've read and re-read the wording and I think that curriculum is covered by "docs education", guides without templates is covered by "content" and other resources is covered by "docs tooling" and "many other initiatives". I'd like to keep it helicopter-level for now.
I totally agree that we are more than templates. Perhaps something that might help is to remove the "first and foremost" wording. I'll leave it in just as a nod to how the project was first formed, way back in 2019.